doc/api/examples/map.json

```diff
@@ -5,6 +5,7 @@
         "desc": "Example map for the API",
         "permission": "commons",
         "screenshot": "https://s3.amazonaws.com/metamaps-assets/site/missing-map.png",
+        "starred": false,
```

Comment:
Ok i can do that

I just dont like changing the schema too much you know? I suppose it might happen more often with these complex fields... 

I was thinking though that as a programmer, its more helpful for this to exist in the response than not exist.

I suppose i could make.its value null. But i actually think i prefer this way